src/codegen/literal/cuda_half_t.h

```diff
@@ -76,7 +77,7 @@ class TVM_ALIGNED(2) half {
   TVM_XINLINE explicit half(const uint8_t& value) { constructor(value); }
   TVM_XINLINE explicit half(const int32_t& value) { constructor(value); }
   TVM_XINLINE explicit half(const uint32_t& value) { constructor(value); }
-  TVM_XINLINE explicit half(const int64_t& value) { constructor(value); }
+  TVM_XINLINE explicit half(const long long& value) { constructor(value); }
```

Comment:
why this change?

Reply:
do I need to `#include <cstdint>` for `int64_t` ?

Reply:
it's okay to use `long long` as they are the same for cuda